### ISSUE TITLE
SpotX 1.7 Premium Installer doesn't installs with new theme.

### ISSUE THREAD

**Issue by ShaggyLorean** [SpotX-Official/SpotX#279]
### 🧐 Have you read the FAQ and viewed similar issues?

- [X] I have read the FAQ
- [X] I have checked older issues, open and closed
- [X] I did a clean install of SpotX as <a href="https://telegra.ph/SpotX-FAQ-09-19#1.-Clean-installation">described in the FAQ</a>

### Сountry of your account

Turkey

### What is your Spotify Plan?

Premium

### Do you have Spicetify installed?

Yes

### ℹ Computer information

```markdown
- Spotify: 1.2.3.1115.gd61a8f5c
- Windows: Windows 11 22H2 25276.1000
- PowerShell: 5.1.2576.1000
```

### 📝 Description

With normal installation named "Install_New_theme.bat" brings new theme but with premium installation named "Install_Prem.bat" brings old theme and not the new theme.

**Comment by amd64fo** [comment]
This new update of Specify won't work with me.

**Comment by amd64fo** [comment]
I hope the author can solve this problem.

**Comment by amd64fo** [comment]
It will show that the base connection has been disconnected and failed to resolve this remote name.This has not happened before.

**Comment by amd64fox** [comment]
[Install_Prem.bat](https://raw.githack.com/SpotX-CLI/SpotX-Win/main/scripts/Install_Prem.bat)

or

```ps1
[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; iex "& { $((iwr -useb 'https://raw.githubusercontent.com/SpotX-CLI/SpotX-Win/main/Install.ps1').Content) } -premium -new_theme"
```

also Spicetify doesn't support left sidebar in 1.2.3, either ask them to fix it or downgrade to 1.2.0
or install old theme:

```ps1
[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; iex "& { $((iwr -useb 'https://raw.githubusercontent.com/SpotX-CLI/SpotX-Win/main/Install.ps1').Content) } -premium"
```
